### PR TITLE
Apiv2

### PR DESCRIPTION
**Pull request by Jo-Inge**

### CODE REVIEW

File: vipps-login-api.md [hunk 1]
```diff
@@ -242,6 +242,7 @@ Vipps Login currently supports the following scopes:
 | phoneNumber | Verified phone number (verfied - the number used with Vipps)                          |   yes   |
 | nin        | Norwegian national identity number (verified with BankID). NB: merchants need to apply for access to NIN. Go to [Who can get access to NIN and how?](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api-faq.md#who-can-get-access-to-nin-and-how) For more information |   yes      |
 | accountNumbers | User bank account numbers. NB: merchants need to apply for access to accountNumbers. Go to [Who can get access to account numbers and how?](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api-faq.md#who-can-get-access-to-accountnumbers-and-how) For more information |   yes      |
+| api_version_2 | Toggle on userinfo version 2 response format. Go to [Vipps Login Api 2.0 migration](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-migrate-api-1.0-to-2.0.md) For more information about migrating from version 1 |   no      |
```
Comment (cloveras) [inline vipps-login-api.md:245]: "API", not "Api"

File: vipps-login-api.md [hunk 2]
```diff
@@ -252,6 +253,10 @@ therefore not accept for example name and deny address.
 We recommend asking for the minimal number of scopes needed for your use case to
 minimize the number of users that deny the consent request.
 
+We recommend all new integrations towards Vipps login to include scope 'api_version_2'.
```
Comment (cloveras) [inline vipps-login-api.md:256]: Login with L

File: vipps-login-api.md [hunk 3]
```diff
@@ -252,6 +253,10 @@ therefore not accept for example name and deny address.
...
+The scope is a toggle between api response version 1 and version 2 of userinfo.
```
Comment (cloveras) [inline vipps-login-api.md:257]: "API", not "api"

File: vipps-login-api.md [hunk 4]
```diff
@@ -242,6 +242,7 @@ Vipps Login currently supports the following scopes:
 | phoneNumber | Verified phone number (verfied - the number used with Vipps)                          |   yes   |
 | nin        | Norwegian national identity number (verified with BankID). NB: merchants need to apply for access to NIN. Go to [Who can get access to NIN and how?](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api-faq.md#who-can-get-access-to-nin-and-how) For more information |   yes      |
 | accountNumbers | User bank account numbers. NB: merchants need to apply for access to accountNumbers. Go to [Who can get access to account numbers and how?](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api-faq.md#who-can-get-access-to-accountnumbers-and-how) For more information |   yes      |
+| api_version_2 | Needs to be added to use the latest version of Vipps login. All merchants need to add this. Go to [Vipps Login API 2.0 migration](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-migrate-api-1.0-to-2.0.md) For more information about migrating from version 1 |   no      |
```
Comment (Jo-Inge) [inline vipps-login-api.md:245]:
```suggestion
| api_version_2 | Needs to be added to use the latest version of Vipps Login. All merchants need to add this. Go to [Vipps Login API 2.0 migration](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-migrate-api-1.0-to-2.0.md) For more information about migrating from version 1 |   no      |
```

**Pull request merged**